### ISSUE TITLE
Usage examples

### ISSUE THREAD

**Issue by MichelML** [eladhoffer/utils.pytorch#11]
Very nice utils!

It'd be cool if you provided examples in action for some/all of them.

I am particularly interested in the cross entropy with label smoothing.

Thanks! #

**Comment by etetteh** [comment]
I tried out the cross-entropy with label smoothing using a smooth value of 0.1, and it works, but I want to know which distribution I can enter in the smooth_dist argument. 
Please, kindly clarify
